go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/asaskevich/govalidator v0.0.0-20200428143746-21a406dcc535
 	github.com/containerd/containerd v1.5.0-rc.3
 	github.com/cyphar/filepath-securejoin v0.2.2
+	github.com/distribution/distribution/v3 v3.0.0-20210804104954-38ab4c606ee3
 	github.com/docker/distribution v2.7.1+incompatible
```

Comment:
Should this be removed?

Reply:
I mentioned this in the OP:

> Once upstream's change has been merged and a new release has been cut, we can drop the dependency on docker/distribution.

We have to wait until oras-go has merged https://github.com/oras-project/oras-go/pull/13 and cut a new release before we can drop this dependency

Reply:
Ok, sorry missed that